dev/devicelab/bin/tasks/module_test_ios.dart

```diff
@@ -184,6 +199,11 @@ dependencies:
         );
       });
 
+      final File mainDart = File(path.join(projectDir.path, 'lib/main.dart'));
+      String mainDartContent = await mainDart.readAsString();
+      mainDartContent = mainDartContent.replaceAll('// UNCOMMENT: ', '');
```

Comment:
TODO @christopherfujino to investigate ensuring that this does not leak state to subsequent tests

Reply:
Ahh, we're doing a `flutter create` into a temp dir on lines 35-48, this is fine :)